src/highlighting.c

```diff
@@ -1436,7 +1436,8 @@ gboolean highlighting_is_string_style(gint lexer, gint style)
 				style == SCE_P_FCHARACTER ||
 				style == SCE_P_FTRIPLE ||
 				style == SCE_P_FTRIPLEDOUBLE ||
-				style == SCE_P_STRINGEOL);
+				style == SCE_P_STRINGEOL ||
+				style == SCE_P_ATTRIBUTE);
```

Comment:
If I understand the usage in LexPython.cxx correctly then this new style is totally not a string, It seems to refer to Python class attributes, i.e. this is more of an identifier like class members.